Source/WebCore/platform/mediastream/webrtcorg/PeerConnectionBackendWebRtcOrg.cpp

```diff
@@ -1,35 +1,54 @@
-#include "config.h"
 #include "PeerConnectionBackendWebRtcOrg.h"
-
-#include "ScriptExecutionContext.h"
-#include "UUID.h"
+#include "Logging.h"
+#include "config.h"
```

Comment:
Could you please manually move config.h to the top of include lists after applying clang-format.